kafka-ui-react-app/src/redux/actions/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { AnyAction } from 'redux';
+import { AppDispatch } from 'redux/interfaces';
+import { alertAdded, alertDissmissed } from 'redux/reducers/alerts/alertsSlice';
+import mockStoreCreator from 'redux/store/configureStore/mockStoreCreator';
+
+export const showSuccessToast = (
```

Comment:
1. We call it Alert. Let's rename it to showSuccessAlert
2. This action should be a part of alertsSlice

Reply:
Renamed it into showSuccessAlert, transformed it into a thunk action in alertsSlice.